### PR TITLE
Nns home page background image

### PR DESCRIPTION
**Pull request by anishh2003**
nns background image has been replaced with an optimised image which is smaller in size

### CODE REVIEW

File: frontend/dart/lib/ui/home/auth_widget.dart [hunk 1]
```diff
@@ -6,7 +6,12 @@ class AuthWidget extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      color: AppColors.lightBackground,
+      constraints: BoxConstraints.expand(),
+      decoration: BoxDecoration(
+        image: DecorationImage(
+            image: AssetImage('assets/nns_background.jpeg'), fit: BoxFit.cover),
+      ),
+      // color: AppColors.lightBackground,
```
Comment (hpeebles) [inline frontend/dart/lib/ui/home/auth_widget.dart:14]: No need to include this commented out line

Comment (anishh2003) [reply frontend/dart/lib/ui/home/auth_widget.dart]: removed this now

File: frontend/dart/lib/ui/home/auth_widget.dart [hunk 2]
```diff
@@ -9,9 +9,8 @@ class AuthWidget extends StatelessWidget {
       constraints: BoxConstraints.expand(),
       decoration: BoxDecoration(
         image: DecorationImage(
-            image: AssetImage('assets/nns_background.png'), fit: BoxFit.cover),
```
Comment (ielashi) [inline frontend/dart/lib/ui/home/auth_widget.dart:12]: We should also delete the old image.

Comment (anishh2003) [reply frontend/dart/lib/ui/home/auth_widget.dart:12]: Ive deleted it now

**Review by hpeebles — APPROVED** [review]

**Pull request merged**